zerver/lib/actions.py

```diff
@@ -3395,13 +3395,13 @@ def user_email_is_unique(email):
         pass
 
 def do_invite_users(user_profile, invitee_emails, streams):
-    # type: (UserProfile, SizedTextIterable, Iterable[Stream]) -> Tuple[Optional[str], Dict[str, List[Tuple[text_type, str]]]]
+    # type: (UserProfile, SizedTextIterable, Iterable[Stream]) -> Tuple[Optional[str], Dict[str, Any]]
     new_prereg_users = [] # type: List[PreregistrationUser]
     errors = [] # type: List[Tuple[text_type, str]]
     skipped = [] # type: List[Tuple[text_type, str]]
 
     ret_error = None # type: Optional[str]
-    ret_error_data = {} # type: Dict[str, List[Tuple[text_type, str]]]
+    ret_error_data = {} # type: Dict[str, Any]
```

Comment:
I assume you changed to Any because you added `bool` there below.

I'd rather go with `Union[bool, List...]` instead of Any - it should be more precise